broker/consumer/lsst/vm_startup.sh

```diff
@@ -57,14 +57,19 @@ fout_topics="${workingdir}/list.topics"
 # define Rubin-related parameters
 kafka_password="${survey}-${PROJECT_ID}-kafka-password"
 KAFKA_PASSWORD=$(gcloud secrets versions access latest --secret="${kafka_password}")
+group_id="pittgoogle-idfint-kafka-pubsub-connector"
+# use test resources, if requested
+if [ "$testid" != "False" ]; then
+    group_id="${group_id}-${testid}"
+fi
 
 cd "${workingdir}" || exit
 
 fconfig=admin.properties
 sed -i "s/KAFKA_PASSWORD/${KAFKA_PASSWORD}/g" ${fconfig}
 
 fconfig=psconnect-worker.properties
-sed -i "s/KAFKA_PASSWORD/${KAFKA_PASSWORD}/g" ${fconfig}
+sed -i "s/KAFKA_PASSWORD/${KAFKA_PASSWORD}/g" ${fconfig} && sed -i "s/GROUP_ID/${group_id}/g" ${fconfig}
```

Comment:
```suggestion
sed -i "s/KAFKA_PASSWORD/${KAFKA_PASSWORD}/g" ${fconfig}
sed -i "s/GROUP_ID/${group_id}/g" ${fconfig}
```

No reason to make the latter dependent on the former.